File: front/src/entity/record/creator.ts
export class Creator {
  name: string;
  secondName: string;
  date: string;
  titles: string;

  getName(): string {
    return this.name;
  }

  setName(value: string) {
    this.name = value;
  }

  getSecondName(): string {
    return this.secondName;
  }

  setSecondName(value: string) {
    this.secondName = value;
  }

  getDate(): string {
    return this.date;
  }

  setDate(value: string) {
    this.date = value;
  }

  getTitles(): string {
    return this.titles;
  }

  setTitles(value: string) {
    this.titles = value;
  }
}

File: back/src/main/java/org/lendi/umtapo/solr/document/bean/record/Creator.java
package org.lendi.umtapo.solr.document.bean.record;

import com.fasterxml.jackson.annotation.JsonInclude;

/**
 * Creator entity.
 */
@JsonInclude(JsonInclude.Include.NON_NULL)
public class Creator {

    private String name;

    private String secondName;

    private String date;

    private String titles;

    /**
     * Gets name.
     *
     * @return the name
     */
    public String getName() {
        return name;
    }

    /**
     * Sets name.
     *
     * @param name the name
     */
    public void setName(String name) {
        this.name = name;
    }

    /**
     * Gets second name.
     *
     * @return the second name
     */
    public String getSecondName() {
        return secondName;
    }

    /**
     * Sets second name.
     *
     * @param secondName the second name
     */
    public void setSecondName(String secondName) {
        this.secondName = secondName;
    }

    /**
     * Gets date.
     *
     * @return the date
     */
    public String getDate() {
        return date;
    }

    /**
     * Sets date.
     *
     * @param date the date
     */
    public void setDate(String date) {
        this.date = date;
    }

    /**
     * Gets titles.
     *
     * @return the titles
     */
    public String getTitles() {
        return titles;
    }

    /**
     * Sets titles.
     *
     * @param titles the titles
     */
    public void setTitles(String titles) {
        this.titles = titles;
    }
}
